CHANGES.rst

```diff
@@ -28,13 +28,17 @@ Improvements
   by users submitting an abstract (:issue:`3138`)
 - Add support for boolean (yes/no) and freetext questions in abstract
   reviewing (:issue:`3175`)
+- Add session types to sessions; allow marking session types as poster
+  so that the sessions with poster session type are displayed differently
+  in the timetable
 
 Internal Changes
 ^^^^^^^^^^^^^^^^
 
-- Rename *Roles* in ACL entries to *Permissions*.  This especially affects
+- Rename *Roles* in ACL entries to *Permissions*. This especially affects
```

Comment:
The double spaces are intentional. In plaintext blobs it's not too uncommon to do this to keep some more spacing between sentences. You can find this in many docstrings or e.g. the GPL license text as well.